.github/sast_configurations/codeql.yml

```diff
@@ -0,0 +1,16 @@
+name: "CodeQL configurations"
```

Comment:
It seems a little odd to me to configure the filters in this repo, since:
-  it's a safe bet that we're gonna want the ability to tune things per project if this gets widely adopted
- with the currently proposed approach, we'd still have to add a YAML file to every repo that wants to use this anyway, yet that file would just be boilerplate and not contain the config

Are we able to override these settings from the repo that calls the workflow?

Reply:
The idea is that developers can add this to their repo as a reusable workflow and the config can still be maintained in a centralized workflow. If it is centralized like this then we are able to easily edit the filters and have them be applied to all the repos using this workflow at the same time. As we mature the process we will want to introduce the lower severity findings and run some of CodeQL's more advanced scans and this should make it easier to edit the filters for everyone simultaneously. 
Allowing tuning on each repo would mean maturing SAST at different paces on each repo rather than doing it synchronously for the entire organization.